scripts/e2e_verify/main.py

```
@@ -0,0 +1,252 @@
+HASH_TIMEOUT = 60 * 1000  # 1 minute
+PRECHECK_START_TIMEOUT = 10 * 60 * 1000  # 10 minutes
+PRECHECK_COMPLETE_TIMEOUT = 4 * 60 * 60 * 1000  # 4 hours
+
+
+def log(step: int, total: int, message: str) -> None:
+    """Print timestamped log message."""
+    timestamp = datetime.now(UTC).strftime("%H:%M:%S")
+    print(f"[{timestamp}] Step {step}/{total}: {message}")  # noqa: T201
+
+
+def run_full_flow(base_url: str, vnc_port: int = 5901) -> bool:  # noqa: PLR0915
```

Ruff (0.14.6)

Unused `noqa` directive (non-enabled: `PLR0915`)

Remove unused `noqa` directive

(RUF100)